.github/pull_request_template.md

```diff
@@ -0,0 +1,57 @@
+### What does this PR do?
+
+<!-- A brief description of the context of this pull request and its purpose. -->
+
+Related to : #<!-- enter issue number here -->
+
+### What are the observable changes?
+<!-- This question could be adequate with multiple use cases, for example: -->
+
+<!-- Frontend: explain the feature created / updated, give instructions telling how to see the change in staging -->
+<!-- Performance: what metric should be impacted, link to the right graphana dashboard for exemple -->
+<!-- Bug: a given issue trail on sentry should stop happening -->
+<!-- Feature: Implements X thrift service / Z HTTP REST API added, provide instructions on how leverage your feature from staging or your workstation -->
+
+### Good PR checklist
+
+- [ ] Title makes sense
+- [ ] Is against the correct branch
+- [ ] Only addresses one issue
+- [ ] Properly assigned
+- [ ] Added/updated tests
+- [ ] Added/updated documentation
+- [ ] Properly labeled
+
+### Impacted components
+
+<!-- List here all the different Upfluence components that could be impacted by your changes -->
+<!-- Please be as precise as possible specifying after the major component which part of it will be impacted -->
+<!-- For example:  -->
+<!-- - [x] Side Panel : Brand Mentions -->
+
+<!-- You must tick at least one component -->
+
+- [ ] CRM :
+- [ ] Mailings :
+- [ ] Workflow :
+- [ ] Search :
+- [ ] Side Panel :
+- [ ] Analytics :
+- [ ] Live Capture :
+- [ ] Payments :
+- [ ] Plugin :
+- [ ] Stores :
+  - [ ] PG store : <!-- Inbox, Analytics, Influencer ... -->
+  - [ ] ES store : <!-- Inbox, Analytics, Influencer ... -->
+  - [ ] Cassandra store : <!-- Activity, Graph-server ... -->
+- [ ] Harvest : <!-- Scrappers -->
+- [ ] Infrastructure : <!-- UDS, Vulcan ... -->
+- [ ] Other : <!-- Your changes concern something more particular -->
+- [ ] No impact <!-- Documentation for example -->
+
```

Comment:
@jlevesy what about adapting for frontend projects ? :)

Reply:
Sure, go ahead ! What do you have in mind ?

Reply:
For this project, I find it relevant. Especially if we want to use it in other projects

Reply:
The only option relevant that could ever be ticked here is CRM. And things like `Stores`, `Harvest`, etc. are hardly reachable from the front 😅 

@jlevesy  For the general frontend case, I would've added the Developer Heads Up about Octane here. (it's just that the pull request template never had

Reply:
@phndiaye @jlevesy  I can add Developer Heads Up about Octane here and change the template with "classic" front template. What do you think?

Reply:
Yes go ahead.